packages/overlay/src/overlay-stack.ts

```diff
@@ -163,10 +182,11 @@ export class OverlayStack {
         if (this.handlingResize) return;
 
         this.handlingResize = true;
-        requestAnimationFrame(() => {
-            this.overlays.forEach((overlay) => {
-                overlay.updateOverlayPosition();
-            });
+        requestAnimationFrame(async () => {
+            const promises = this.overlays.map((overlay) =>
+                overlay.updateOverlayPosition()
+            );
+            await Promise.all(promises);
             this.handlingResize = false;
```

Comment:
@Westbrook This is where I had the `for of` loop. I suspect that I did that because I was going to `await` the popper resizes. You can do that in a `for of`, but not in a `forEach`. The only other advantage of `for of` is that it avoids the overhead of a function call.

However, I realized that would serialize the updates (not that there should be many poppers open at a time). I changed it to do the updates in parallel.